### ISSUE TITLE
Can't reproduce precision on pascal_voc

### ISSUE THREAD

**Issue by kakaxi314** [jwyang/faster-rcnn.pytorch#100]
I'm sorry that I tried 3 times, but I can't get 74.9 map which is listed in your table.
I use command 
`python trainval_net.py --dataset pascal_voc --net res101 --bs 4 --nw 4 --lr 4e-3 --lr_decay_step 8 --epochs 10 --cuda`
I barely changed the code and got map of 72.4,72.9,73.5 based on caffe model as pretrain.
I want to know how can I get 74.9 map for resn101?

**Comment by jwyang** [comment]
Hi, @kakaxi314, did you use the pretrained model pointed by the link in the readme?

**Comment by kakaxi314** [comment]
@jwyang Of course, I downloaded the pretrained model from you link. I want to know whether my command is wrong or anything in code I need to change?

**Comment by jwyang** [comment]
@kakaxi314 can you reproduce the performance using vgg16?

**Comment by kakaxi314** [comment]
@jwyang Hi，I trained VGG16 by using command
`python trainval_net.py --dataset pascal_voc --net vgg16 --bs 1 --nw 4 --lr 1e-3 --lr_decay_step 5 --epochs 7 --cuda`
I got 69.1 map, not 70.2 as your list.
I want to know is it normal?  Otherwise, is the command not right?

**Comment by jwyang** [comment]
@kakaxi314 Hi, which epoch did you evaluate? I think the mAP is a bit lower if you set batch size =1

**Comment by jwyang** [comment]
@kakaxi314 i am training the model on my side, will let you know when it finishes

**Comment by kakaxi314** [comment]
@jwyang I almost evaluated models of all the epoches , but none of them gets similar mAP as yours.  Meanwhile, by using your trained model, I can get same mAP, which means the _test_net.py_ works well. Thanks for your concern and wait for your reply.

**Comment by CupidJay** [comment]
Would you be so kind to tell me where could I download the author's trained-model as you mentioned above? I want to use the trained-model to test on other datasets. Thank you very much.

**Comment by jwyang** [comment]
@kakaxi314 Hi, I trained vgg16 on my side with the same command as yours, and one epoch 6, I got:

AP for aeroplane = 0.7055
AP for bicycle = 0.7761
AP for bird = 0.6638
AP for boat = 0.5457
AP for bottle = 0.5221
AP for bus = 0.8086
AP for car = 0.8448
AP for cat = 0.8411
AP for chair = 0.5025
AP for cow = 0.7819
AP for diningtable = 0.6539
AP for dog = 0.7754
AP for horse = 0.8294
AP for motorbike = 0.7267
AP for person = 0.7726
AP for pottedplant = 0.4223
AP for sheep = 0.7188
AP for sofa = 0.6555
AP for train = 0.7526
AP for tvmonitor = 0.7277
Mean AP = 0.7014

the mAP is 70.1 on my side. Everything looks fine.

**Comment by kakaxi314** [comment]
I retrained several times, but 69.2 is the best accuracy I can get.
Last time, in  one epoch 6, I got:
AP for aeroplane = 0.7398
AP for bicycle = 0.7872
AP for bird = 0.6665
AP for boat = 0.5396
AP for bottle = 0.4810
AP for bus = 0.7644
AP for car = 0.8442
AP for cat = 0.8459
AP for chair = 0.4151
AP for cow = 0.7680
AP for diningtable = 0.6091
AP for dog = 0.8030
AP for horse = 0.8347
AP for motorbike = 0.7601
AP for person = 0.7652
AP for pottedplant = 0.3716
AP for sheep = 0.6554
AP for sofa = 0.6419
AP for train = 0.7360
AP for tvmonitor = 0.6605
Mean AP = 0.6845

I can't find  the reason for this performance gap.
Do you use Pytorch 0.2.0?
If so, I will try to change my pytorch 0.3.1 to pytorch 0.2.0.

**Comment by jwyang** [comment]
@kakaxi314 yes, I used 0.2.0, I guess it is the initialization that might affect the performance.
